style/base.scss

```diff
@@ -69,8 +71,8 @@
 	--string-color: #a2db3c;
 	--font-size: 11px;
 	--input-font-size: 11px;
-	--font-family: -apple-system, BlinkMacSystemFont, 'Lucida Grande', 'Segoe UI', Roboto, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
```

Comment:
I forgot what boilerplate these definitions came from. They don't need to be this long.